templates/package/content/arch.tmpl

```diff
@@ -0,0 +1,105 @@
+{{if eq .PackageDescriptor.Package.Type "arch"}}
+	<h4 class="ui top attached header">{{.locale.Tr "packages.installation"}}</h4>
+	<div class="ui attached segment">
+		<div class="ui form">
+
+			<div class="field">
+				<label>{{svg "octicon-note"}} {{.locale.Tr "packages.arch.pacmanconf"}}</label>
+				<div class="markup"><pre class="code-block"><code>[{{.PackageDescriptor.Owner.LowerName}}.{{.PackageDescriptor.Metadata.BaseDomain}}]
+Server = https://{{.PackageDescriptor.Metadata.BaseDomain}}/api/packages/arch/(distribution)/(architecture)/{{.PackageDescriptor.Owner.LowerName}}</code></pre></div>
+			</div>
+
+			<div class="field">
+				<label>{{svg "octicon-clock"}} {{.locale.Tr "packages.arch.pacmansync"}}</label>
+				<div class="markup"><pre class="code-block"><code>pacman -S {{.PackageDescriptor.Package.LowerName}}</code></pre></div>
+			</div>
+
+			<div class="field">
+				<label>{{.locale.Tr "packages.arch.documentation" "https://docs.gitea.io/en-us/usage/packages/arch/" | Safe}}</label>
+			</div>
+		</div>
+	</div>
+
+	<h4 class="ui top attached header">{{.locale.Tr "packages.arch.properties"}}</h4>
+	<div class="ui attached segment">
+	<table class="ui single line very basic table">
+		<tbody>
+
+			<tr>
+				<td class="collapsing"><h5>Description</h5></td>
+				<td>{{.PackageDescriptor.Metadata.Description}}</td>
+			</tr>
+			<tr>
+				<td class="collapsing"><h5>Compressed size</h5></td>
+				<td>{{.PackageDescriptor.Metadata.CompressedSizeMib}}</td>
+			</tr>
+			<tr>
+				<td class="collapsing"><h5>Installed size</h5></td>
+				<td>{{.PackageDescriptor.Metadata.InstalledSizeMib}}</td>
+			</tr>
+			<tr>
+				<td class="collapsing"><h5>Official URL</h5></td>
+				<td> <a target="_blank" rel="noopener noreferrer" href="{{.PackageDescriptor.Metadata.URL}}">{{.PackageDescriptor.Metadata.URL}}</a></td>
+			</tr>
+			<tr>
+				<td class="collapsing"><h5>Build date</h5></td>
+				<td>{{.PackageDescriptor.Metadata.BuildDateStr}}</td>
+			</tr>
+			<tr>
+				<td class="collapsing"><h5>Packager</h5></td>
+				<td>{{.PackageDescriptor.Metadata.Packager}}</td>
+			</tr>
```

Comment:
This should be in the sidebar.

Reply:
Can i leave URL in the same place? I might not fit into sidebar...

Reply:
Just have a look how it is done for the other package types. The url is not shown, just something like [Project Site](https://gitea.com)

Reply:
![Screenshot from 2023-06-22 11-38-10](https://github.com/go-gitea/gitea/assets/58640116/f2cc613b-f823-4d78-9110-032117bc4f95)
Move URL to side bar, removed build date, compressed and installed size.
Packager does not fit into side bar (it has name and email), would it be ok to leave it on the same place?